include/sys/util.h

```diff
@@ -363,6 +363,23 @@ static inline uint8_t bin2bcd(uint8_t bin)
  */
 uint8_t u8_to_dec(char *buf, uint8_t buflen, uint8_t value);
 
+/**
+ * @brief Copy src to dest word-wise
+ *
+ * This function copies src to dest 32 bits at a time.
+ * If one calls this function without src and dst being aligned
+ * on a 4-byte boundary or if n is not a multiple of 4, it will
+ * do nothing and return NULL.
```

Comment:
Reading this I thought it was weird there was no error-handling, but I see there are __ASSERT's.

These should be mentioned here.